containers/elm/.devcontainer/Dockerfile

```diff
@@ -3,40 +3,36 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 
-FROM mcr.microsoft.com/vscode/devcontainers/base:debian-9
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:12
 
 # Avoid warnings by switching to noninteractive
 ENV DEBIAN_FRONTEND=noninteractive
 
 # Configuring Elm version
-ARG ELM_VERSION=0.19.0
+ARG ELM_VERSION=latest-0.19.1
+ARG ELM_TEST_VERSION=latest-0.19.1
+ARG ELM_FORMAT_VERSION=latest-0.19.1
 
 # This Dockerfile's base image has a non-root user with sudo access. Use the "remoteUser"
 # property in devcontainer.json to use it. On Linux, the container user's GID/UIDs
 # will be updated to match your local UID/GID (when using the dockerFile property).
 # See https://aka.ms/vscode-remote/containers/non-root-user for details.
-ARG USERNAME=vscode
+ARG USERNAME=node
```

Comment:
I changed the username from `vscode` to `node` because that's what is used in [`javascript-node:12`](https://github.com/microsoft/vscode-dev-containers/blob/cd4e2d538d15efc1f39c160bd106d83fbe28e8e3/containers/javascript-node-12/.devcontainer/Dockerfile#L15). Is this correct?

Reply:
Yep!  That's exactly right.